packages/sui-js/test/pipeSpec.js

```diff
@@ -4,7 +4,7 @@ import pipe from '../src/pipe/index'
 
 describe('@s-ui/js', () => {
   const textToUpperCase = text => text.toUpperCase()
-  const textToArray = text => [...text]
+  const textToArray = text => text.split('')
```

Comment:
And change this functions, for others: https://github.com/SUI-Components/sui/commit/fd8df3b27f017b5d790063a060a662d3cbf2d355

But I have to say that your fix is more elegant.